### ISSUE TITLE
Compiling for windows fails

### ISSUE THREAD

**Issue by shuttle1987** [JaggedVerge/mmap_backed_array#5]
After setting up the MSVC compiler with http://landinghub.visualstudio.com/visual-cpp-build-tools the package can't install.
Attempting to install this package via pip will fail with the following message:
```
_cffi__x8e0c389ax618b51a5.c
.tox\py35\lib\site-packages\mmap_backed_array\__pycache__\_cffi__x8e0c389ax618b51a5.c(209): fatal error C1083: Cannot op
en include file: 'sys/mman.h': No such file or directory
```

Now this is somewhat as expected due to the dependency on a header that's missing in Windows. The question is if this is worth working around or not as this library has core assumptions about mmap-ing of memory being efficient on linux. Being able to support windows would be dependent on having some form of native fast-memory allocation on windows.

**Comment by shuttle1987** [comment]
Perhaps this: https://github.com/witwall/mman-win32

**Comment by shuttle1987** [comment]
Fixed with #6
